docs/src/main/asciidoc/azure-support-overview-concept.adoc

```diff
@@ -0,0 +1,117 @@
+////
+This document is maintained in the main Quarkus repository
+and pull requests should be submitted there:
+https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
+////
+[id="azure-support-overview-concept"]
+= Quarkus on Microsoft Azure cloud platform
+include::_attributes.adoc[]
+:categories: cloud
+
+Quarkus provides support and capabilities to help you to build cloud-native applications that run on Microsoft Azure. Quarkus also provides support for building serverless Java functions by using Microsoft Azure Functions.
+You can deploy the container image of a Quarkus application that you have developed locally to the Microsoft Azure cloud platform.
+You can also entirely build, host, and run a Quarkus application on Azure by using some of the key developer services for Azure.
+
+== Microsoft Azure
+
+link:https://learn.microsoft.com/en-us/azure/?product=popular[Microsoft Azure] is a versatile and flexible cloud platform, which you can use to host your Quarkus applications.
+Azure supports various programming languages, complex architectures and frameworks, including Java and Quarkus.
+Azure also fully supports the Apache Maven Java development tool.
+You can use several link:https://learn.microsoft.com/en-us/azure/developer/intro/azure-developer-key-services[Azure key services for developers] to build and extend your Quarkus applications.
+You can also integrate and deploy Quarkus with the serverless environment of link:https://learn.microsoft.com/en-us/azure/azure-functions/[Azure Functions].
+
+Microsoft Azure also provides tools and services that integrate with Quarkus to make it easier to develop, deploy, and manage your Java applications on its cloud platform. For example:
+
+* link:[Azure DevOps] provides CI/CD pipelines for building and deploying Quarkus applications.
+* link:[Azure Monitor] provides capabilities for you to use to monitor and troubleshoot these applications.
+
+== Quarkus extensions for Microsoft Azure
+
+Quarkus provides the following extensions to make it easier to integrate your existing Quarkus applications with Microsoft Azure:
+
+
+* `quarkus-azure-functions`
+* `quarkus-azure-functions-http`
+
+// What about these extensions? Should we mention them?
```

Comment:
These are different from the funky ones in the sense these are providing integration with specific azure cloud services. 

And yes; if this guide is about what you can do with quarkus in azure it makes sense to mention them.